libs/vkd3d/device.c

```diff
@@ -2138,6 +2138,33 @@ void d3d12_device_return_scratch_buffer(struct d3d12_device *device, const struc
     }
 }
 
+uint64_t d3d12_device_get_descriptor_heap_gpu_va(struct d3d12_device *device)
+{
+    uint64_t va;
+
+    /* The virtual GPU descriptor VAs are of form (unique << 32) | (desc index * sizeof(d3d12_desc)),
```

Comment:
Can anything bad happen if these overlap with resource VAs?

Reply:
There is no way to use these VAs in any other context than as a table, so I don't see how.

Reply:
Technically we could always just return 0 for the top bits, but tools like apitrace will probably get very confused.